Objects/Defines.swift

```diff
@@ -0,0 +1,7 @@
+
+import Foundation
+import UIKit
+ struct Defines {
```

Comment:
Should change Defines to Constants. Constants will closely meaning when we declare only constants. Defines is everything that determined, not only constants.